src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+
+  &__star {
+    display: inline-block;
+    width: 16px;
```

Comment:
It is better to use variables for widths and heights, and margins for better maintainability. Consider creating a variable for the star size and using it for width, height, and margin-right.